src/edu/stanford/dlss/wowza/SulWowza.java

```diff
@@ -333,4 +333,18 @@ HttpURLConnection getStacksHttpURLConn(URL stacksUrl, String requestMethod)
         stacksConn.setReadTimeout(stacksReadTimeout * 1000);  // need milliseconds
         return stacksConn;
     }
+
+    static String urlEncode(String urlComponent)
+    {
+        try
+        {
+            // the javadocs say that encode without an explicitly specified encoding is deprecated
```

Comment:
I would leave this comment out;  you are doing the right thing here.  That's enough.